ext/phar/phar_internal.h

```diff
@@ -88,6 +88,8 @@
 #define PHAR_SIG_SHA256           0x0003
 #define PHAR_SIG_SHA512           0x0004
 #define PHAR_SIG_OPENSSL          0x0010
+#define PHAR_SIG_OPENSSL_256      0x0011
```

Comment:
Here and elsewhere, I think these should be called `PHAR_SIG_OPENSSL_SHA256`.

Reply:
done